src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1302 @@
+    edidver = (tv_hdmi_edid_version_t)select;
+        
+    ret = dsSetEdidVersion(port , edidver);
+    UT_LOG_INFO("Result : Passed dsSetEdidVersion port type: %s, portnumber: %s\t"
+                         "selected edid type: %s, edid version:%d\t",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port), port,
+                 UT_Control_GetMapString(tv_hdmi_edid_version_mapTable, edidver) , edidver);
+    assert(ret == dsERR_NONE);
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to get edidversion.
+*
+* This test function is to get edidversion of HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 012@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_edidversion(void)
+{
+    gTestID = 12;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    int32_t select = 0;
+    dsHdmiInPort_t port = dsHDMI_IN_PORT_NONE;
+    tv_hdmi_edid_version_t edidver = HDMI_EDID_VER_MAX;
+    uint8_t numInputPorts = 0;
+
+    numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/numberOfPorts");
+    UT_LOG_INFO("\n**********Please select port***********\n");
+
+    readInput(&select);
+    if(select <= 0 || select >(numInputPorts-1))
+    {
+      UT_LOG_ERROR("\n invalid port selected\n");
+    }
+    port = (select - 1);
+   
+    ret = dsGetEdidVersion(port, &edidver);
+    UT_LOG_INFO("Result : dsGetEdidVersion port type: %s, portnumber: %s\t"
+                         "retrieved edid type: %s, edid version:%d",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port), port,
+                 UT_Control_GetMapString(tv_hdmi_edid_version_mapTable, edidver) , edidver);
+    assert(ret == dsERR_NONE);
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to get allmstatus.
+*
+* This test function is to get allmstatus of HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 013@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_allmstatus(void)
+{
+    gTestID = 13;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    int32_t select = 0;
+    dsHdmiInPort_t port = dsHDMI_IN_PORT_NONE;
+    bool allmstatus = false;
+    uint8_t numInputPorts = 0;
+
+    numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/numberOfPorts");
+    UT_LOG_INFO("\n**********Please select port***********\n");
+
+    readInput(&select);
+    if(select <= 0 || select > (numInputPorts -1))
+    {
+      UT_LOG_ERROR("\n invalid port selected\n");
+    }
+    port = (select - 1);
+
+    ret = dsGetAllmStatus(port , &allmstatus);
+    UT_LOG_INFO("Result : Passed dsGetEdidVersion port type: %s, portnumber: %s\t"
```

Comment:
Dont add as "Result: Passsed" only add as Result . Also Add in IN , OUT details before and after the fucntion

Reply:
done